react/Label/Readme.md

```diff
@@ -0,0 +1,11 @@
+#### Label element for forms
+
+```
+<Label for="id">This is a label</Label>
```

Comment:
I see that the prop name is `forID` so I think there is a typo here. I prefer the name `for` to stay as close as possible to the HTML property.

Reply:
Would be great to have an example with an input inside so that the `for` property is tested.

Reply:
Well I wasn't sure if it was a reserved keyword honestly, that's why I used `forID`.

Reply:
As for the input exemple, because I'm still writing it, I'll add it later to the example.

Reply:
Yup that's what I thought, linter doesn't like it.
`Syntax error: for is a reserved word`

Reply:
React has `htmlFor`: https://reactjs.org/docs/dom-elements.html#htmlfor